transmission_interface/test/simple_transmission_loader_test.cpp

```diff
@@ -0,0 +1,382 @@
+// Copyright 2022 PAL Robotics S.L.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "transmission_interface/simple_transmission_loader.hpp"
+#include <pluginlib/class_loader.hpp>
+
+#include <gtest/gtest.h>
```

Comment:
We are usually using gmock? Would this be possible?

```suggestion
#include <gmock/gmock.h>
```